news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+﻿---
+layout: post
+title: "Seasonal Spotlights: Fall 2018"
+date: 2018-12-31 06:00:00 +0000
+---
+
+Although the end of 2018 is approaching fast we still have some unopened presents left for this year. Come and take a look at the best maps of the Fall Season and earn a fancy new medal for yourself!
+
+![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/header.jpg)
+
+The beatmap charts for the Fall Spotlights can be found [here](https://osu.ppy.sh/rankings/osu/charts?spotlight=252). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Fall 2018 Spotlights rankings will end on the 15th of March, 2019.**
+
+## Summer Spotlights results
+
+First of all, congratulations to the winners of the Seasonal Spotlights: Summer 2018. They receive 3, 2, and 1 months of supporter for 1st, 2nd, and 3rd place respectively for their efforts!
++ **osu!:** [Toy](https://osu.ppy.sh/users/2757689), [StreaminDemon](https://osu.ppy.sh/users/6245906) and [emu1337](https://osu.ppy.sh/users/2185987)
++ **osu!taiko:** [Jaye](https://osu.ppy.sh/users/4841352), [apaajaboleh10](https://osu.ppy.sh/users/5151647) and [BluePlusSymbol](https://osu.ppy.sh/users/6503700)
++ **osu!catch:** [RAMPAGE88](https://osu.ppy.sh/users/448547), [Crystal](https://osu.ppy.sh/users/1646397) and [Nelly](https://osu.ppy.sh/users/4741164)
++ **osu!mania:** [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), [robby250](https://osu.ppy.sh/users/2653437) and [Cryolien](https://osu.ppy.sh/users/1626983)
+
+# Navigation
+
+- <a href="#osu">osu!</a>
+- <a href="#taiko">osu!taiko</a>
+- <a href="#catch">osu!catch</a>
+- <a href="#mania">osu!mania</a>
+
+# Fall 2018 Spotlights
+
+## <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/kaoru.jpg)](https://osu.ppy.sh/beatmapsets/397764)
+
+Contrasting her mapset of "Odoru Mizushibuki", [Hollow Wings](https://osu.ppy.sh/users/416662) brings us a very different interpretation of this melody line in her mapset of the sister arrangement Akiyama Uni's "Kaoru Juyouka". The set is designed to shrink in circle size for each difficulty level, with each difficulty showing off different design concepts for each circle size, accumulating in a wonderful yet challenging Circle Size 7 low-bpm stream challenge in the highest difficulty. Careful attention to detail was placed on the stream designs to reflect the various pitches of the melody line, making for a very unique experience nobody should miss.
```

Comment:
low-bpm  -> low BPM
on line 121 there is "high BPM", so this should be consistent.